app/models/rent.rb

```diff
@@ -1,5 +1,5 @@
 class Rent < ApplicationRecord
-  belongs_to :user
-  belongs_to :book
+  belongs_to :user, optional: true
```

Comment:
Humm, why this change?

Reply:
Active admin didn't let me create Rents without that change. It kept giving me an error, I think it was "must exist" or something like that.